packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useBreadcrumbsStyleProps } from './useBreadcrumbsStyleProps';
+import { SpiritBreadcrumbsProps } from '../../types';
+import { useStyleProps } from '../../hooks/styleProps';
+
+const defaultProps = {
+  items: [],
+};
+
+export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcrumbsProps<T>): JSX.Element => {
+  const { elementType: ElementTag = 'nav', ...restProps } = props;
+  const { classProps, props: modifiedProps } = useBreadcrumbsStyleProps(restProps);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  return (
+    <ElementTag
+      {...otherProps}
+      {...styleProps}
+      className={classNames(classProps, styleProps.className)}
+      aria-label="Breadcrumb"
+    >
+      <ol>
+        {restProps.items.map((item, index) => (
+          <React.Fragment key={`BreadcrumbsItem_${item.url}`}>
```

Comment:
Are you sure that `item.url` is a great candidate for React `key` prop? Especially with all those weird strings `//:?&@`... I would suggest using `item.title` as a better candidate. What do you think?

Reply:
Well I tried to use `index` since there shouldn't be any re-rendering issues within this component but eslint told me not to do that. OK, I changed `item.url` to `item.title`. I think that breadcrumbs should not contain any item that matches url nor title anyway.